devicetypes/smartthings/ecolink-wireless-siren.src/ecolink-wireless-siren.groovy

```diff
@@ -68,13 +68,17 @@ def parse(String description) {
 		result = zwaveEvent(cmd)
 		
 		if(result!=null) {
-			createEvent(result)
+			//device sends 'OK' message when is plugged in
+			if(!result.contains('OK')) {
+				createEvent(result)
+			}
 		}
 	}
 }
 
 def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd) {	
 	createEvents(cmd.value)
+	sendHubCommand(refresh())
```

Comment:
you should change the order of these, as you've inadvertently changed the return value

I'm also concerned that by not restricting this to only refresh the children that you have created an infinite loop